radio/src/options.h

```diff
@@ -100,6 +100,9 @@ static const char * const options[] = {
 #endif
 #if defined(BIND_KEY)
   "bindkey",
+#endif
+#if defined(HARDWARE_TOUCH)
```

Comment:
Would prefer not to display this, as this would not be a firmware option

Reply:
ok

Reply:
not sure though why it wouldn't be an option ... I mean, you would compile it into any boards?

Reply:
I mean we would include that by default in all radio that support touch, so this would not be an option (option are the tick boxes that you have in companion)

Reply:
got it